.github/workflows/ci.yml

```diff
@@ -96,3 +96,29 @@ jobs:
         timeout_minutes: 15
         max_attempts: 2
         command: make ci-integration
+
+  blst-test:
```

Comment:
Looks great! Could we introduce a periodic build as a [scheduled event](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events) to get signal?

Reply:
From what I read about the schedules, it seems to be possible to schedule the entire workflow, but not a specific job. Did you mean to create a new workflow for the blst tests?

Reply:
I'm actually fine with running this job step as an out-of-matrix job for now, we can actually move on to dealing with it as a separate workflow if we get further issues from CI.

Reply:
Do you mean to leave the job as it is in the PR for now? 
I'm also looking at how to make the job not required for merging PRs but I don't seem to find the GH settings. Any idea how to set this up?

Reply:
> Do you mean to leave the job as it is in the PR for now?
Yes

> I'm also looking at how to make the job not required for merging PRs but I don't seem to find the GH settings. Any idea how to set this up?


It's a positive setting in status checks for the repo:
https://docs.github.com/en/github/administering-a-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-status-checks-before-merging

Reply:
@Kay-Zee I think I would need your help to set this new job to not required to merge PRs 🙏